specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1251,6 +1251,108 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/topology": {
+            "get": {
+                "tags": ["Topology"],
+                "description": "Gets a list that allows to build a toplogy view of subscription.",
+                "operationId": "Toplogy_List",
+                "parameters": [{
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/TopologyList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{ascLocation}/topology": {
+            "get": {
+                "tags": ["Topology"],
+                "description": "Gets a list that allows to build a toplogy view of subscription and location.",

```

Comment:
Gets a list that allows to build a **topology** view of **a** subscription and location.